ops/solana/solana.go

```diff
@@ -180,21 +220,80 @@ func (d Deployer) TransferLINK() error {
 
 func (d Deployer) InitOCR(keys []map[string]string) error {
 
-	jsonKeys, err := json.Marshal(keys)
+	fmt.Println("Setting up OCR Feed:")
+
+	fmt.Println("Begin set config...")
+	err := d.gauntlet.ExecCommand(
+		"ocr2:begin_offchain_config",
+		d.gauntlet.Flag("network", d.network),
+		d.gauntlet.Flag("version", "1"),
+		d.gauntlet.Flag("state", d.States[OCRFeed]),
+	)
+	if err != nil {
+		return errors.New("begin OCR 2 set config failed")
+	}
+
+	S := []int{}
+	offChainPublicKeys := []string{}
+	peerIDs := []string{}
+	for _, k := range keys {
+		S = append(S, 1)
+		offChainPublicKeys = append(offChainPublicKeys, k["OCROffchainPublicKey"])
+		peerIDs = append(peerIDs, k["P2PID"])
+	}
+
+	input := map[string]interface{}{
+		"deltaProgressNanoseconds": 2 * time.Second,
+		"deltaResendNanoseconds":   5 * time.Second,
+		"deltaRoundNanoseconds":    1 * time.Second,
+		"deltaGraceNanoseconds":    500 * time.Millisecond,
+		"deltaStageNanoseconds":    5 * time.Second,
+		"rMax":                     3,
+		"s":                        S,
+		"offchainPublicKeys":       offChainPublicKeys,
+		"peerIds":                  peerIDs,
+		"reportingPluginConfig": map[string]interface{}{
+			"alphaReportInfinite": false,
+			"alphaReportPpb":      uint64(1000000),
+			"alphaAcceptInfinite": false,
+			"alphaAcceptPpb":      uint64(1000000),
+			"deltaCNanoseconds":   15 * time.Second,
+		},
+		"maxDurationQueryNanoseconds":                        2 * time.Second,
+		"maxDurationObservationNanoseconds":                  2 * time.Second,
+		"maxDurationReportNanoseconds":                       2 * time.Second,
+		"maxDurationShouldAcceptFinalizedReportNanoseconds":  2 * time.Second,
+		"maxDurationShouldTransmitAcceptedReportNanoseconds": 2 * time.Second,
+	}
+
+	jsonInput, err := json.Marshal(input)
 	if err != nil {
 		return err
 	}
 
+	fmt.Println("Writing set config...")
+	err = d.gauntlet.ExecCommand(
+		"ocr2:write_offchain_config",
+		d.gauntlet.Flag("network", d.network),
+		d.gauntlet.Flag("state", d.States[OCRFeed]),
+		d.gauntlet.Flag("input", string(jsonInput)),
+	)
+
+	if err != nil {
+		return errors.New("writing OCR 2 set config failed")
+	}
+
+	fmt.Println("Committing set config...")
 	err = d.gauntlet.ExecCommand(
-		"ocr2:set_config:deployer",
+		"ocr2:commit_offchain_config",
 		d.gauntlet.Flag("network", d.network),
-		d.gauntlet.Flag("keys", string(jsonKeys)),
 		d.gauntlet.Flag("state", d.States[OCRFeed]),
 	)
 
 	if err != nil {
-		return errors.New("OCR 2 set config failed")
+		return errors.New("committing OCR 2 set config failed")
```

Comment:
`errors.Wrap(...` please, and wrap the original error so we get more information out.